### DISCUSSION TITLE
Nas or different backup

### DISCUSSION THREAD

**Discussion by levent-levent** [bluecherrydvr/bluecherry-apps#420] [Q&A]
Hello,

i have bluecherry and about 60 camera. i am using dvr. but  i can not record on windows pc or another backup pc.. it is only on the bluecherry dvr server recording. 
How can i record camera another backup device, or record windows or different backup linux system

Check please attachment photo,

Regards.
![blue1](https://user-images.githubusercontent.com/77454625/104622257-f0c64a00-5688-11eb-9a4d-32d85741d615.PNG)

**Comment by sicada** [comment]
As that error message shows, bluecherry is expecting that path to be (or behave like) a local path (commands like mkdir and chmod can't work directly on remote paths / hosts). 

To write directly to remote hosts, you have to employ a network file system (NFS, CIFS, SMB/SAMBA or similar), which will allow you to transparently map a local path on the bluecherry server to a path on some remote server. There are many online guides that provide step-by-step setup examples for setting up such a service.

Be aware of sync and cache settings for networked filesystems, as bluecherry server says on the page in your screenshot:  "It is recommended to mount network storage with sync option and disabled cache (option cache=none for CIFS)."
